spec/test-data/translations/ios_l10n_helper/expected-merged-prefixed.strings

```diff
@@ -0,0 +1,34 @@
+/* Generated File. Do not edit. */
+
+/* MARK: - Localizable-utf16.strings */
+
+/* Some file header comment here */
+
+/* comment on key 1 */
+"main.key1" = "string 1";
+
+/* comment on key 2 */
+"main.key2" = "string 2️⃣";
+
+/* multiline comment on key 3.
+   %d = number of lines
+   and = a red herring for when merge_strings tries to add prefixes to lines it identifies as pairs using unquoted keys */
```

Comment:
Notice the traps to ensure the code adding prefixes to keys does not accidentally add prefixes to those lines (and unexpectedly transform them to `"main.%d" = number of lines` or `"main.and" = a red herring …`)